mlflow/server/auth/__init__.py

```diff
@@ -178,6 +180,25 @@ def _get_permission_from_experiment_id() -> Permission:
     )
 
 
+_EXPERIMENT_ID_PATTERN = re.compile(r"^(\d+)/")
```

Comment:
What if the user's file has a bunch of numbers in it? Just making sure that this works in this case; worth adding a test :)

Reply:
Thanks for the comment, sounds good, I'll add a test for that!